RSHighscores.php

```diff
@@ -20,10 +20,13 @@
 $wgExtensionCredits['parserhook'][] = array(
 	'path'			=> __FILE__,
 	'name'			=> 'RSHiscores',
-	'version'		=> '3.0.1-dev',
+	'version'		=> '3.1.1-dev',
 	'descriptionmsg'	=> 'rshiscores-desc',
 	'url'			=> 'https://github.com/TehKittyCat/RSHiscores',
-	'author'		=> '[http://runescape.wikia.com/wiki/User_talk:TehKittyCat TehKittyCat]',
+	'author'		=> array(
+		'[http://runescape.wikia.com/wiki/User_talk:TehKittyCat TehKittyCat]',
+		'[http://runescape.wikia.com/wiki/User:Cqm Cqm]',
+	),
```

Comment:
Probably should add Quarenon as well, he did give design feedback and the initial draft of the caching functionality.